torchao/swizzle/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
we don't want to create a new folder under torchao for this tensor/op I think..

Reply:
Where do you recommend for it to go?

Reply:
is this prototype? we can add to torchao/prototype for now